content/appendices/templates.md

```diff
@@ -47,20 +47,30 @@ The package includes all the following forms of documentation:
 Readme requirements
 The package meets the readme requirements below:
 
-- [ ] Package has a README.md file in the root directory. 
+- [ ] Package has a user-friendly README.md file in the root directory. 
 
 The README should include, from top to bottom:
 
 - [ ] The package name
 - [ ] Badges for continuous integration and test coverage, the badge for pyOpenSci peer-review once it has started (see below), a repostatus.org badge, and any other badges. If the README has many more badges, you might want to consider using a table for badges, see this example, that one and that one. Such a table should be more wide than high.
-- [ ] Short description of goals of package, with descriptive links to all vignettes (rendered, i.e. readable, cf the documentation website section) unless the package is small and there’s only one vignette repeating the README.
+- [ ] Short description of goals of package
+- [ ] Dscriptive links to all vignettes (rendered, i.e. readable, cf the documentation website section) unless the package is small and there’s only one vignette repeating the README.
 - [ ] Installation instructions
 - [ ] Any additional setup required (authentication tokens, etc)
 - [ ] Brief demonstration usage
 - [ ] Direction to more detailed documentation (e.g. your documentation files or website).
 - [ ] If applicable, how the package compares to other similar packages and/or how it relates to other packages
 - [ ] Citation information
 
+#### Usability
+Reviewers are encouraged to submit suggestions (or pull requests) that will improve the usability of the package as a whole. 
+Package structure should follow general community best-practices. In general please consider:
+
+- [ ] The documentation is easy to find and understand
+- [ ] The need for the package is clear
+- [ ] All functions have documentation and associated examples for use
```

Comment:
How do you feel about suggesting/requiring the use of the [sphinx coverage extension](https://www.sphinx-doc.org/en/master/usage/extensions/coverage.html)?

Reply:
I'm usually -1 to required specific tools. I'm definitely +1 to recommend them.
I'm of an opinion that, asking too much from a new scientist/developer may drive the person away. IMO a single script deployed with `flit` on PyPI is a good candidate for PyOpenSci.

Reply:
i agree that requiring specific extensions isn't ideal. ropensci also agrees. For coverage they just ask for reasonable test coverage and then an explanation associated with coverage numbers that are low. They do not specify requirements for docs coverage. we are using autodoc and doctest to cover doc tests and probably wouldn't want to switch given that works for us!!

Reply:
+1 on not requiring specific extensions. 

I would also suggest breaking this item into two 
- [ ] Example use
- [ ] API documentation

As they might live in different parts of the docs, and I think we want to encourage reviewers to explicitly look for both. Also, it for examples, it might make more sense to have a single example that uses multiple functions rather than one example per function (and the current language suggests we want one-per function). 

One last point, I quite like the JOSS `Good`, `OK`, `Bad (not acceptable)` breakdown for items like the API docs, installation instructions, and testing: https://joss.readthedocs.io/en/latest/review_criteria.html#api-documentation. It might be worth thinking about hosting a similar page for pyOpenSci to give reviewers and editors a few decision criteria  when questions of "where is the bar" come up.